### DISCUSSION TITLE
failed to load plugin ''quickadd''

### DISCUSSION THREAD

**Discussion by rubyame** [chhoumann/quickadd#747] [Q&A] [answered]
Hello I installed this plugin first time but I can't load it. My current version is v1.7.4, I need some help please

**Comment by cjbarroso** [comment]
I have the same error. Fresh install and fresh vault, when I try to activate it says it cannot load it.
Attached console logs:

[quiadd-faillog.log.log](https://github.com/user-attachments/files/17542825/quiadd-faillog.log.log)

**Comment by MCXopjesh** [comment]
I'm getting the same error, here is the info I have.

Windows 10

Obsidian v1.7.4
Installer v1.6.7
Quickadd v1.11.1

All of which are up to date at the time of writing this on 11/01/2024

The error occurs any time I attempt to enable the plugin. If I try to use the toggle switch, the switch will not flip. If i use the enable button on the plugin page, the toggle switch will flip but it still will not work.

I attempted to install QuickAdd on a fresh vault and restarted my computer, but the error persists.

Looking into the code, I get this error message:

Plugin failure: quickadd DOMException: Failed to execute 'structuredClone' on 'Window': W=>b(d=>({...d,...W})) could not be cloned.
    at i5 (plugin:quickadd:55:5868)
    at async GI.onload (plugin:quickadd:73:7765)

That's the most I could find out

**Comment by chhoumann** [comment, marked as answer]
Hey, I'm sorry for taking so long about getting this fixed! It should hopefully be resolved now. I hope you enjoy QuickAdd, and please let me know if there are any further issues. All the best, Christian.

**Reply by chhoumann** [reply]
See https://github.com/chhoumann/quickadd/issues/728

**Comment by chhoumann** [comment]
Closing the loop on this one: the failure was a `structuredClone` crash during plugin load on Obsidian 1.7.4 (issue #728), fixed in November 2024. Current QuickAdd loads fine.

If anyone hits a load failure on a current version, please open a new issue with the console log so I can take a look.
